src/events_dispatcher.js

```diff
@@ -20,6 +20,19 @@
     return this;
   };
 
+  prototype.bind_pattern = function(pattern, userCallback){
+    if (!(pattern instanceof(RegExp))) {
+      Pusher.warn("::bind_pattern expects a RegExp");
+      return;
+    }
```

Comment:
people might expect to be able to pass a string in here which would be passed to `new RegExp` ? Wdyt?